cloc

```diff
@@ -10338,6 +10341,14 @@ sub set_constants {                          # {{{1
                                 [ 'smarty_to_C'                    ],
                                 [ 'call_regexp_common'  , 'C'      ],
                             ],
+    'Snakemake'          => [
+                                [ 'remove_matches'      , '/\*'    ],
```

Comment:
I'm guessing the `remove_matches` using `'/\*'` and `'\*/'` were intended to capture C style comments?  That's not what will happen, instead these will strip only from these markers to the end of the line.  Just remove these two lines (10345 & 10346) and you'll be fine.   `/*` .. `*/` comments are captured with `call_regexp_common` `C`.